libzeth/serialization/proto_utils.tcc

```diff
@@ -43,15 +43,24 @@ zeth_proto::HexPointBaseGroup2Affine point_g2_affine_to_proto(
     const libff::G2<ppT> &point)
 {
     assert(!point.is_zero());
-    using Fq = libff::Fq<ppT>;
     libff::G2<ppT> aff = point;
     aff.to_affine_coordinates();
+    const std::vector<std::string> x_coord =
+        ext_field_element_to_hex<libff::Fqe<ppT>>(aff.X);
+    const std::vector<std::string> y_coord =
+        ext_field_element_to_hex<libff::Fqe<ppT>>(aff.Y);
```

Comment:
(Related to the comments about generic functions over fields, and tests over multiple curves). Will this work for bw6-761?